code/game/objects/items/weapons/his_grace.dm

```diff
@@ -130,7 +130,7 @@
 	gender = MALE
 	adjust_bloodthirst(1)
 	force_bonus = HIS_GRACE_FORCE_BONUS * LAZYLEN(contents)
-	playsound(user, 'sound/effects/pope_entry.ogg', 100)
+	playsound(user, 'sound/effects/his_grace_awaken.ogg', 100)
```

Comment:
This doesn't actually sound that good/imposing, it's why I switched away from it in the first place.